--- /usr/share/misc/termcap	2017-07-29 16:46:57.370274000 +0000
+++ termcap	2017-08-09 03:09:03.753635000 +0000
@@ -2883,7 +2883,7 @@
 
 xterm-256color|xterm alias 3:\
 	:Co#256:pa#32767:\
-	:AB=\E[48;5;%dm:AF=\E[38;5;%dm:tc=xterm-new:
+	:tc=xterm-new:
 
 xterm-nrc|xterm alias 4:\
 	:tc=xterm:
@@ -4636,7 +4636,7 @@
 
 rxvt-256color|rxvt terminal emulator with 256 colors:\
 	:Co#256:pa#32767:\
-	:AB=\E[48;5;%dm:AF=\E[38;5;%dm:op=\E[39;49m:tc=rxvt-unicode:
+	:AF=\E[3%dm:AB=\E[4%dm:op=\E[39;49m:tc=rxvt-unicode:
 
 rxvt-unicode-256color|rxvt-unicode terminal with 256 colors (X Window System):\
 	:tc=rxvt-256color:
